chcemvediet/apps/inforequests/tests/test_models/test_inforequest.py

```diff
@@ -892,6 +949,26 @@ def test_branch_by_pk_method_with_invalid_pk(self):
             with self.assertRaises(ValueError):
                 self.assertEqual(inforequest.branch_by_pk(47), branch2)
 
+    def test_create_classmethod(self):
+        inforequest = Inforequest.create(
+            applicant=self.user,
+            subject=u'Subject',
+            content=u'Content',
+            obligee=self.obligee,
+            subject_finalize=lambda inforequest: u'Subject Finalize',
+            content_finalize=lambda inforequest: u'Content Finalize',
+        )
+        self.assertEqual(inforequest.branch_set.count(), 0)
+        inforequest.save()
+        self.assertEqual(inforequest.branch_set.count(), 1)
+
+    def test_get_absolute_url_method(self):
+        inforequest = self._create_inforequest()
+        lang = ((u'sk', u'Slovak'), (u'en', u'English'))
+        for language_code, _ in lang:
+            with translation(language_code):
+                self.assertEqual(inforequest.get_absolute_url(u'anchor'), u'/{}/{}/{}-{}/anchor'.format(language_code, __(u'main:urls:inforequests'), inforequest.slug, inforequest.pk))
```

Comment:
Ked je to taky dlhy riadok, mozno najprv do premennej prirad `expected_url` a rozdel to na dva riadky, alebo to nejak zalom.

Reply:
:ok: